src/vector/app.tsx

```diff
@@ -124,7 +124,7 @@ function onTokenLoginCompleted() {
     parsedUrl.search = "";
```

Comment:
Maybe it's better to just clean particular query param?
In our case it's the `loginToken`